src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/CollectionModulesConsistency.fs

```diff
@@ -8,11 +8,17 @@ open NUnit.Framework
 open FsCheck
 open Utils
 
+/// helper function that creates labeled FsCheck properties for equality comparisons
+let consistency name sqs ls arr =
+    (sqs = arr) |@ (sprintf  "Seq.%s = %A, Array.%s = %A" name sqs name arr) .&. 
```

Comment:
can you put ticks around the %A I.e. '%A'  it allows us to see that a value is missing in the event of bugs.